applications/GeoMechanicsApplication/custom_utilities/transport_equation_utilities.hpp

```diff
@@ -156,6 +157,8 @@ class GeoTransportEquationUtilities
         return result;
     }
 
+    [[nodiscard]] static double CalculateParticleDiameter(const Properties& rProperties);
```

Comment:
As the transport equation utilities are for the Pw parts of our elements, this indeed is a suitable place to have this equation ( and reuse it from here ).